examples/basic/index.html

```diff
@@ -95,6 +95,9 @@
         clearInterval(interval);
       }
     });
+    setTimeout(function(){
```

Comment:
Maybe rather than doing a `setTimeout`, we can just set the value to 1 in the markup so that it is spinning from the start?

Reply:
Well I can change this to static, but I the reason I prefered to delay the start was mostly to show how to change attribute dynamically from javascript... this is where that update method is relevant... so I thought this example will illustrate more the real use.

For the context I plan to use widgets also in read only mode and change them from external events, eg:
http://github.com/SamsungInternet/color-sensor-js

Reply:
I see. Just don't want anyone confused about why the value changed if they don't look at the code.  Maybe one possibility is to hook up one of the buttons so that it toggles the switch? A bit of labeling using `<a-text>` would go a long way as well.  See https://aframe.io/docs/0.9.0/primitives/a-text.html

Given that the repo is pretty old, i'm going to bump the a-frame version up to something more current.  Hopefully that won't break anything for you?

Reply:
yea this is what i am actually doing with my code ... 

I have tested current aframe it seems to work , but I need to sort out how to use slider with mouse or gearvr2 controller..

Reply:
Mmm. Yeah, mouse-click and pan is difficult since it's also used for panning/rotating the scene. Any ideas?